kvirt/k3s/kcli_default.yml

```diff
@@ -1,48 +1,51 @@
 info: |
-    Deploys the k3s kubernetes distrobution using an arbitrary number of masters and workers.
```

Comment:
you just reorder the variables here? cos i have a hard time figuring out what was added based on the github diff

Reply:
Yeah ... I thought that this little brush-up could make you a little agitated. All I'm trying to-do is make the file more readable. And finding 'x' variable when the file is in lexicographical order just finds me easier.
And I added the three K3s arg. array variables.